src/Detector/Hugo/HugoDetector.cs

```diff
@@ -66,31 +66,43 @@ private bool IsHugoApp(ISourceRepo sourceRepo, out string appDirectory)
             appDirectory = string.Empty;
 
             // Search for config.toml
-            if (sourceRepo.FileExists(HugoConstants.TomlFileName)
-                && this.IsHugoTomlFile(sourceRepo, HugoConstants.TomlFileName))
+            foreach (string tomlFileName in HugoConstants.TomlFileName)
             {
-                return true;
+                if (sourceRepo.FileExists(tomlFileName)
+                && this.IsHugoTomlFile(sourceRepo, tomlFileName))
```

Comment:
nit: let's indent the second line of each if-statement so that it follows the previous format

```cs
if (sourceRepo.FileExists(tomlFileName)
    && this.IsHugoTomlFile(sourceRepo, tomlFileName))
```

This comment applies for the other instances of this below.